### ISSUE TITLE
Module not found: Error: Can't resolve '@microsoft/load-themed-styles'

### ISSUE THREAD

**Issue by nguyentoantuit** [OfficeDev/office-ui-fabric-react#6042]
### Bug Report 

- __Package version(s)__: 6.52.0
- __Browser and OS versions__: Windows 10
 
#### Describe the issue:
 
I have just update to latest version and run into this issue 

 ```
ERROR in ./~/@uifabric/styling/lib/styles/theme.js
    Module not found: Error: Can't resolve '@microsoft/load-themed-styles' in 'D:\SWGLawyer\KolektoHuur\Kolekto.Huur.CreditWeb\node_modules\@uifabric\styling\lib\styles'
     @ ./~/@uifabric/styling/lib/styles/theme.js 6:0-77
     @ ./~/@uifabric/styling/lib/styles/index.js
     @ ./~/@uifabric/styling/lib/index.js
     @ ./~/@uifabric/icons/lib/fabric-icons.js
     @ ./~/@uifabric/icons/lib/index.js
     @ ./ClientApp/boot-client.tsx

    ERROR in ./~/office-ui-fabric-react/lib/components/Panel/Panel.scss.js
    Module not found: Error: Can't resolve '@microsoft/load-themed-styles' in 'D:\SWGLawyer\KolektoHuur\Kolekto.Huur.CreditWeb\node_modules\office-ui-fabric-react\lib\components\Panel'
     @ ./~/office-ui-fabric-react/lib/components/Panel/Panel.scss.js 2:0-59
     @ ./~/office-ui-fabric-react/lib/components/Panel/Panel.js
     @ ./~/office-ui-fabric-react/lib/components/Panel/index.js
     @ ./~/office-ui-fabric-react/lib/Panel.js
     @ ./ClientApp/components/topNavigation/appBar.tsx
     @ ./ClientApp/components/topNavigation/topBar.tsx
     @ ./ClientApp/app.tsx
     @ ./ClientApp/routes.tsx
     @ ./ClientApp/boot-client.tsx
```

I have tried to install @microsoft/load-themed-styles but it doesn't work. 
Are there any one have this issue before and how can you fix it. I saw few reports #3812 #3818.

**Comment by austin-best** [comment]
I am experiencing this as well.

**Comment by nguyentoantuit** [comment]
@austin-best there is a problem with release of @microsoft/load-themed-styles 1.7.80. It is fixed in version 1.7.81.
The full story is here: #782

**Comment by realhoffi** [comment]
the npm folder does not contain a lib or a lib-amd folder! So you have to run the following commanf...

npm install --no-save @microsoft/load-themed-style

**Comment by realhoffi** [comment]
Still got this issue! :) It was fixed some time ago, but now its there again :(

**Comment by joechung-msft** [comment]
Fixed in 1.7.81 and broken again in 1.7.82.

**Comment by kenotron** [comment]
I believe this is the cause: https://github.com/Microsoft/web-build-tools/issues/796 - load-themed-styles is published by another team within MSFT; and it seems to have been addressed there.
